src/components/navbar/NavBar.module.scss

```diff
@@ -0,0 +1,130 @@
+@font-face {
+    font-family: lotrFont;
+    src: url(../../fonts/Middleearth-ao6m.ttf);
+  }
+
+  .navBar_outer_wrapper {
+    position: sticky;
+		top: 0;
+  }
+
+  .navBar_inner_wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    height: fit-content;
+    max-width: 100vw;
+    color: #162027;
+    font-family: lotrFont;
+    background: url(../../images/navbar.jpg);
+    box-shadow: 5px 5px 8px #888888;
+  }
+
+  .appBar {
+    display: flex;
+    flex-grow: 3;
+    width: 350px;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    //border: 1px solid green;
+  }
+
+  .appBar__item {
+    text-decoration: none;
+    color: #c4bb9a;
+    text-align: center;
+    border-radius: 10px;
+    // padding: 0 10px 0 10px;
+
+    & button {
+      border-radius: 10px;
+    }
+  }
+
+  .appBar__item:hover {
+    cursor: pointer;
+    // background-color: rgb(#c4bb9a, 0.5);
+    // max-width: max-content;
+    // border-radius: 10px;
+    // padding: 0 10px 0 10px;
+    color: #a6cfeb ;
+  }
+
+  .appBar__search {
+    flex-grow: 0.6;
+    height: 45px;
+    width: 180px;
+    justify-items: center;
+    border-radius: 10px;
+    border: none;
+    opacity: 0.5;
+    //border: 2px solid blue;
+    text-align: left;
+    margin: 10px 0 10px 0;
+    opacity: 0.9; 
+    padding-left: 40px;
+  }
+
+  .appBar__search:focus {
+    background-color: #a6cfeb;
+  }
+
+  .appBar__search::placeholder {
+    text-align: left;
+  }
+
+  .appBar__search_wrapper {
+    position: relative;
+   
+    &::after {
+      content: '';
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      right: 150px;
+      margin: auto 0;
+      width: 20px;
+      height: 20px;
+      background: url('../../images/magnifying-glass.png') no-repeat;
+      background-size: cover;
+    }
+  }
+
+  span {
+    margin-left: 20px;
+    margin-right: 20px;
+    text-align: center;
+    color: #c4bb9a;
+  }
+
+  .appBar_logout {
+    flex-grow: 1;
+    display: flex;
+    align-items: center;
+    justify-content: end;
+    padding-right: 20px;
+    //flex-direction: column;
+    //border: 2px solid yellowgreen;
+
+    & button {
+      width: 90px;
+      align-self: center;
+      margin-bottom: 5px;
+    }
+
+    & .appBar__logout__btn {
+      margin-right: 15px;
+    
+      & button {
+        border-radius: 10px;
+      }
+
+      & button:hover {
+        cursor: pointer;
+        background-color: #a6cfeb;
+      }
+    }
+
+
```

Comment:
LGFM, but next time try to use rem instead of px, it gives you free adaptivity in most cases